CoreFoundation/Base.subproj/CFPriv.h

```diff
@@ -90,7 +90,7 @@ CF_EXPORT void CFPreferencesFlushCaches(void);
 CF_EXPORT Boolean _CFURLGetWideFileSystemRepresentation(CFURLRef url, Boolean resolveAgainstBase, wchar_t *buffer, CFIndex bufferLength);
 #endif
 
-#if !__LP64__
+#if !__LP64__ && !__LLP64__
```

Comment:
Given the prevalence o this, I’m beginning to suspect a _CF_64_BIT macro may not be a bad idea.

Reply:
Hehe, yes, I really did want that previously too :-).

Reply:
Maybe switch to `__POINTER_WIDTH__` like the Swift runtime?

Reply:
I think that would work. There are probably still places where `LP64` means a few different things (e.g., historically some have used it to indicate presence of modern runtime, incorrectly, on Darwin), but I've tried to fix those up as I find them.

Reply:
Or, I guess, `TARGET_RT_64_BIT` from `TargetConditionals.h` below, which @compnerd fixed up as well.

Reply:
I think I ended up favoring @parkera's suggestion of `TARGET_RT_64_BIT` as it fits more inline with the rest of the project.  Split into #1808